src/app/space/create/deployments/apps/deployment-details.component.ts

```diff
@@ -126,8 +132,8 @@ export class DeploymentDetailsComponent {
           this.netVal = round(netTotal.used, 1);
           this.netUnits = netTotal.units;
           this.netData.xData.push(+new Date());
-          this.netData.yData[0].push(stat.sent.raw);
-          this.netData.yData[1].push(stat.received.raw);
+          this.netData.yData[0].push(stat.sent.used);
+          this.netData.yData[1].push(stat.received.used);
```

Comment:
IIRC I used `.raw` here rather than `used` to deal with the frequent changes in units and scaling that can occur with network traffic, ex. traffic on the order of bytes in one sample and on the order of kilo or megabytes in the next. Using `.used`, 800 bytes looks hundreds of times larger than 1KB.